android/cli/commands/_buildModule.js

```diff
@@ -1607,7 +1607,7 @@ AndroidModuleBuilder.prototype.packageZip = function (next) {
 			moduleJarArchive.on('error', cb);
 			moduleJarArchive.pipe(moduleJarStream);
 
-			var excludeRegex = /.*\/R\.class$|.*\/R\$(.*)\.class$/i;
+			var excludeRegex = new RegExp('.*\\' + path.sep + 'R\\\.class$|.*\\' + path.sep + 'R\\\$(.*)\\\.class$', 'i');
```

Comment:
The three backslashes as escape characters are unnecessary here, for example `R\\.class$` already results in in the correct regex `R\.class$`